CMakeModules/FindGMock.cmake

```diff
@@ -0,0 +1,129 @@
+# Locate the Google C++ Mocking Framework.
```

Comment:
I have very little familiarity with cmake, was the compile broken before adding this?

Reply:
it wasn't broken since we install gmock on the VM. but this module allows clion to find gmock similar to gtest. less red code